packages/contracts/src/resources/site.ts

```diff
@@ -530,12 +548,19 @@ export const siteApi = contract.router(
 		},
 	},
 	{
+		strictStatusCodes: true,
```

Comment:
this makes the return types from the api client more legible. instead of getting 
```ts
type Response = 
| { status: 200, body: {...}} 
| { status: 201 | 202 | ...., body: any} 
```

you get

```ts
type Response = { status: 200, body: {...}} 
```